apps/condo/domains/organization/schema/RegisterNewOrganizationService.js

```diff
@@ -26,6 +27,11 @@ const RegisterNewOrganizationService = new GQLCustomSchema('RegisterNewOrganizat
                 const adminRole = defaultRoles.Administrator
                 await createConfirmedEmployee(context, organization, context.authedItem, adminRole, extraData)
                 await createTrialSubscription(context, organization, extraData)
+                const contextWithoutAccessControl = await context.createContext({ skipAccessControl: true })
+                await TicketOrganizationSetting.create(contextWithoutAccessControl, {
```

Comment:
you don't need to create a new context. Server utils already have skip access

Reply:
pls resolve conversation only after you push changes